src/lib/s3Uploader.test.js

```diff
@@ -70,3 +70,18 @@ describe('s3Uploader', () => {
     await expect(got).toBe(response);
   });
 });
+
+describe('s3Uploader.getPresignedUrl', () => {
+  const Bucket = 'fakeBucket';
+  const Key = 'fakeKey';
+  const mockGetURL = jest.spyOn(s3, 'getSignedUrl').mockImplementation(async () => ({ url: 'https://example.com' }));
+  beforeEach(() => {
+    mockGetURL.mockClear();
+  });
+  it('calls getSignedUrl() with correct parameters', async () => {
+    const url = getPresignedURL(Key, Bucket);
+    await expect(url).resolves.toMatchObject({ url: 'https://example.com' });
+    expect(mockGetURL).toHaveBeenCalled();
+    expect(mockGetURL).toHaveBeenCalledWith('getObject', { Bucket, Key, Expires: 360 });
+  });
+});
```

Comment:
Good test!
Could we also add one for an error condition?

Reply:
yes we can!